floodviz/static/js/analytics_helper.js

```diff
@@ -0,0 +1,45 @@
+(function (i, s, o, g, r, a, m) {
+	i['GoogleAnalyticsObject'] = r;
+	i[r] = i[r] || function () {
+			(i[r].q = i[r].q || []).push(arguments)
+		}, i[r].l = 1 * new Date();
+	a = s.createElement(o),
+		m = s.getElementsByTagName(o)[0];
+	a.async = 1;
+	a.src = g;
+	m.parentNode.insertBefore(a, m)
+})(window, document, 'script', 'https://www.google-analytics.com/analytics.js', 'ga');
+
+ga('create', 'UA-103831764-1', 'auto');
+ga('send', 'pageview');
+
+
+// GA for DOM content not related to figures.
+document.addEventListener('DOMContentLoaded', function (event) {
+
+	// Social Media Click events for GA
+	var sm_icons = document.getElementsByClassName('icon');
+	Array.prototype.forEach.call(sm_icons, function (element) {
```

Comment:
Is there a reason you are using Array.prototype.forEach.call rather than sm_icons.forEach? This is also one place where I think it's clearer to add a 'onclick' property to each icon in index.html: so something like 
<img src="static/images/faceblook.pn" onclick="ga_send_event('Social_Media', 'click_icon', 'Facebook')". Then this code can be eliminated